### ISSUE TITLE
"completed-docs" lint option is not detected on the IDE 

### ISSUE THREAD

**Issue by jhm-ciberman** [Microsoft/vscode-tslint#307]
- tslint Version: 5.8.0
- typescript Version: 2.6.1
- vscode-tslint version: 1.0.24

Steps to Reproduce:

1. Add this to the tslint.json file in the `rules` section:
```
"completed-docs": [
    true, 
   "classes", 
   "functions",
   "methods", 
   "properties"
]
```
2. Create a new ts file with any class, function, method or property without JSDoc documentation
3. Lint the file from the console with `$ tslint` to verify that the linter detects the ausence of proper JSDoc documentation.
4. Open the ts file in atom with vscode-tslint active, it will *not* display the linter error.  (It should)

**Comment by egamma** [comment]
This rule requires type information and these rules are currently not supported please see the discussion in #70. 

Closing as a duplicate of #70.
